src/components/o-commercial-hero-cover/index.scss

```diff
@@ -0,0 +1,115 @@
+@import "../../styles/typo/typo-mixins";
+@import "../../styles/settings/colors";
+@import "../../styles/grid/grid-mixins";
+@import "../../styles/mixins/respond";
+@import "../../styles/typo/typo-mixins";
+@import "../../styles/mixins/word-break";
+@import "../m-button/mixins/button";
+
+.o-commercial-hero-cover {
+  display: block;
+
+  .o-commercial-hero-cover__header {
```

Comment:
i like that the bem classes works only in the main block element, but this can lead to the following problem:

If a user wants to override some styling, he also need to have the same specificty + 1 which would lead to unneded code and more difficulty in maintaing the code.

Therefore, my suggestion is to not put the elements inside the block and relay on naming to avoid collision and pollution